lms/static/sass/views/_login-register.scss

```diff
@@ -67,7 +67,7 @@ $sm-btn-linkedin: #0077b5;
         }
     }
 
-    form {
+    form, .hint-wrapper {
```

Comment:
When adding structural or styling-minded wrapper elements, we add the ``wrapper-`` prefix. Please change to ``wrapper-hint``.